cmd/src/users_prune.go

```diff
@@ -68,43 +68,50 @@ query {
 			return err
 		}
 
-		usersQuery := `
-query Users() {
-	users() {
-		nodes {
-			...UserFields
-		}
-	}
+		getInactiveUsersQuery := `
+query getInactiveUsers {
+	  site {
+    users {
+      nodes {
+        username
+        email
+        siteAdmin
+        lastActiveAt
+      }
+    }
+  }
```

Comment:
Intendation is off

Reply:
This is hitting the `aggregated_user_statistics` table, and the [value](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/cmd/frontend/graphqlbackend/site_users.go?L91-97&subtree=true) of `lastActiveAt` doesn't need to access the `event_logs` table the same way `usageStatistics` used to. Try it out [here](https://k8s.sgdev.org/api/console#%7B%22query%22%3A%22query%20InactiveUsersShort%20%7B%5Cn%20%20site%20%7B%5Cn%20%20%20%20users%20%7B%5Cn%20%20%20%20%20%20nodes%20%7B%5Cn%20%20%20%20%20%20%20%20id%5Cn%20%20%20%20%20%20%20%20username%5Cn%20%20%20%20%20%20%20%20email%5Cn%20%20%20%20%20%20%20%20siteAdmin%5Cn%20%20%20%20%20%20%20%20lastActiveAt%5Cn%20%20%20%20%20%20%7D%5Cn%20%20%20%20%7D%5Cn%20%20%7D%5Cn%7D%22%7D), its much more performant.

`aggregated_user_statistics` table is [updating in the background](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@84726dc70827092c17547b6d7a35c505d8834895/-/blob/internal/users/update_aggregated_stats_job.go), so this isn't accessing the event logs table at the time of the request.

Reply:
https://github.com/sourcegraph/src-cli/pull/903/files Indentation fix